### PR TITLE
[WIP] Trying some different options for a PEP8-like style reference

### PR DESCRIPTION
**Pull request by willingc**
[rendered guide landing page](http://test-black.readthedocs.io/en/style-guide/style_guide/style_guide.html)

1. [one possible style for content page](http://test-black.readthedocs.io/en/style-guide/style_guide/line_wrap.html)
2. [another option](http://test-black.readthedocs.io/en/style-guide/style_guide/guards.html)

Goals:
- more visual than text
- examples (input and Black) displayed
- make it easy to reference back to PEP 8

### CODE REVIEW

**Comment by ambv** [comment]
I like the "Guards" example better. We can keep the conversational-style intro extracted from the README as "Code style overview" but concrete examples like you did in "Guards" are making things very clear with minimal text. I like this a lot.

**Comment by willingc** [comment]
Guards and examples it is. Thanks for making the direction straightforward.

File: docs/style_guide/guards.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+def spaces_types(
+    a: int = 1,
+    b: tuple = (),
+    c: list = [],
+    d: dict = {},
+    e: bool = True,
+    f: int = -1,
+    g: int = 1 if False else 2,
+    h: str = "",
+    i: str = r'',
```
Comment (zsol) [inline docs/style_guide/guards.md]: nit: *Black* actually formats this line into
```
    i: str = r"",
```

There's one more example of this down below, where `'w'` should be replaced with `"w"`

**Comment by willingc** [comment]
@ambv I've been swamped in a research grant proposal. Hoping to free up at the end of the week. :-)

File: docs/style_guide/empty_lines.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Empty lines
+
+## Emit extra empty line after control flow statements
+
+*Black* will always emit an extra empty line after ``return``, ``raise``,
```
Comment (chmreid) [inline docs/style_guide/empty_lines.md]:
```suggestion
*Black* will always emit an extra empty line after `return`, `raise`, 
```

File: docs/style_guide/empty_lines.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Empty lines
+
+## Emit extra empty line after control flow statements
+
+*Black* will always emit an extra empty line after ``return``, ``raise``, 
+``break``, ``continue``, and ``yield``.
```
Comment (chmreid) [inline docs/style_guide/empty_lines.md]:
```suggestion
`break`, `continue`, and `yield`.
```

**Comment by chmreid** [comment]
@willingc is this PR still relevant? Can help as needed

**Comment by cooperlees** [comment]
This PR has become very stale. Please reopen if you wish to finish it.

**Pull request closed without merging**